src/Analyzer.Core.BuiltInRuleTests/Tests/SQL_Rentention_Days.json

```diff
@@ -0,0 +1,22 @@
+﻿[
+  {
+    "Template": "SQLServerAuditingSettings",
+    "ReportedFailures": [
+      // The resource between lines 10 and 33 passes this check because all the elements of the AnyOf are true
+      // The ones between lines 34 and 113 passes because at least one of the elements of the AnyOf is true
+      // And the ones between lines 135 and 173 passes because of empty scopes
+      {
+        "LineNumber": 127,
+        "Description": "isAzureMonitorTargetEnabled set to false"
+      },
+      {
+        "LineNumber": 128,
+        "Description": "storageEndpoint is not an empty string"
+      },
+      {
+        "LineNumber": 129,
+        "Description": "retentionDays set to a value different than 0 and less than 90"
```

Comment:
I think this could be phrased differently? Would be it more clear to say that retentionDays was set to a value greater than 0 and less than 90? I guess this wouldn't work if negative retention days is a valid template value though, do you know if that would even pass validation?

Reply:
I wrote it like that because the rule specifies that retentionDays should equal to 0 or be greater than 90. Considering that, do you think is still confusing?

Reply:
Yeah I think "equal to 0 or greater than 90" sounds more clear than the inverse "different than 0 or less than 90" because there's some implicit overlap between different than 0 and less than 90, whereas the conditions equal to 0 or greater than 90 overlap. At least that's how my mind processes it? So imo I think "retentionDays **not** set to a value equal to 0 or greater than 90" is easier to comprehend, also since it's the simple negation of what the expectation is.

Reply:
There's technically some wiggle room to argue that "retentionDays not set to a value equal to 0 or greater than 90" means:
!(retentionDays == 0) || retentionDays > 90
but I still read it as 
!(retentionDays == 0 || retentionDays > 90)